tools/genassets.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python2
+#!/usr/bin/env python3
```

Comment:
This change is in the wrong commit. Please combine all "/usr/bin/env python3" changes into a single commit, together.